index.css

```diff
@@ -0,0 +1,171 @@
+html body {
+    font-family: Helvetica, sans-serif;
+    font-size: 22px;
+    margin: 0;
+    padding: 0;
+    background-color: black;
+    color: rgba(255, 246, 240, 0.9);
+}
+
+nav {
+    display: inline-flex;
+    position: fixed;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: solid rgb(255, 245, 238, 0.9) 1px;
+    height: 69px;
+    width: 100%;
+    background-color: black;
+    top: 0;
+}
+
```

Comment:
Very generic slectors like this can/will cause complexity issues down the line. I would reccomend learning the [BEM naming method](https://css-tricks.com/bem-101/) and using it instead.